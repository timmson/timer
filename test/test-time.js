const React = require("react");
const renderer = require("react-test-renderer");

const Time = require("../src/time");

class Moment {

	format(format) {
		return (format === "HH:mm:ss, dddd, DD.MM.YYYY Ð³.") ? "xxx" : null;
	}
}

function moment() {
	return new Moment();
}

describe("Time should", () => {

	test("return formatted time", () => {
		const component = renderer.create(
			<Time moment={moment()} calendarURL={"url"}/>
		);
		expect(component.toJSON()).toMatchSnapshot();
		component.unmount();
	});

});