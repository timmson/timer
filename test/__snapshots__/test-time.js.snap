// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Time should return formatted time 1`] = `
<div
  className="row"
>
  <div
    className="col currentTime"
  >
    <a
      href="url"
    />
  </div>
</div>
`;
